Simplified/en.lproj/Localizable.strings

```diff
@@ -223,6 +223,8 @@
 "SigningInAgree" = "By signing in, you agree to the End User License Agreement.";
 "Want a card for your child?" = "Want a card for your child?";
 "Cannot confirm library card eligibility." = "Cannot confirm library card eligibility.";
+"New patron sign up is not supported by this library." = "New patron sign up is not supported by this library.";
+"Sign up process fails to initiate." = "Sign up process fails to initiate.";
```

Comment:
The wording for these error messages need to be reviewed by @killerpuppytails,
and also need translation from @ettore after.

Reply:
for the time being here's the italian translations for the current strings (I will fix them once we get the approved wording from Risa):

"New patron sign up is not supported by this library." = "La creazione di nuovi utenti non è supportata da questa biblioteca";
"Sign up process fails to initiate." = "Il process di registrazione non è potuto iniziare.";